src/vs/workbench/contrib/chat/browser/chatEditing/chatEditingSession.ts

```diff
@@ -250,15 +251,18 @@ export class ChatEditingSession extends Disposable implements IChatEditingSessio
 	}
 
 	private _getEntry(uri: URI): ChatEditingModifiedDocumentEntry | undefined {
-		return this._entriesObs.get().find(e => isEqual(e.modifiedURI, uri));
+		const notebookUri = CellUri.parse(uri)?.notebook;
+		return this._entriesObs.get().find(e => isEqual(e.modifiedURI, uri) || isEqual(e.modifiedURI, notebookUri));
```

Comment:
This prevents having to modify all calling code to always deal with Notebook Cell Uris as well.
Not sure this is the right thing, but seems to make sense.
E.g. if you are in a cell, using the Cell Uri one can get hold of the Entry associated with the notebook.

Reply:
👍 tho, I pushed a simplification to make it `uri = CellUri.parse(uri)?.notebook ?? uri;` because we should never have an edits session for just a cell, it's either for notebook or file-backed documents

Reply:
Yes that makes sense, wanted to keep that as `inline chat` for notebooks was relying on the Cell Uris having its own  ModifeidEntry. 
Then again, none of the old `inline chat notebook` would go through edits, so its ok.